cli/src/commands/account/chooseMember.ts

```diff
@@ -0,0 +1,43 @@
+import AccountsCommandBase, { ISelectedMember } from '../../base/AccountsCommandBase'
+import chalk from 'chalk'
+import { flags } from '@oclif/command'
+import ExitCodes from '../../ExitCodes'
+
+export default class AccountChoose extends AccountsCommandBase {
+  static description = 'Choose default account to use in the CLI'
+  static flags = {
+    address: flags.string({
```

Comment:
Done - flag changed from `address` to `memberId`. 

I was reworking `AccountChoose` from Sumer release since something like that was missing in Giza and mistakenly considered address as 1st class citizen here, but it should be a member, as you pointed out.